src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -280,6 +289,11 @@
       --#{$menu-toggle}--m-secondary--before--BorderColor: var(--#{$menu-toggle}--m-expanded__toggle--m-secondary--before--BorderColor);
       --#{$menu-toggle}--m-secondary--before--BorderWidth: var(--#{$menu-toggle}--m-expanded__toggle--m-secondary--before--BorderWidth);
     }
+
+    &:disabled,
+    &.pf-m-disabled {
+      --#{$menu-toggle}--BackgroundColor: var(--#{$menu-toggle}--disabled--BackgroundColor);
+    }
```

Comment:
As far as I can tell, this block isn't doing anything. Do you know if it's needed for any particular variation/use of the secondary toggle?

Reply:
You are correct, it's not needed after switching the `hover` `focus` `active` and `expanded` variables to be explicitly set.